src/scss/design-system-interface.scss

```diff
@@ -167,8 +171,8 @@ $c-pale: #F5F1F6;
 }
 
 .drizzle-p {
-  font-size: 18px;
-  font-family: 'Roboto', sans-serif;
+  font-size: 1.125rem;
```

Comment:
We could also consider creating a utility for font-sizes.. for example `font-size(sm)` (xs, md, lg, xl etc). Heres a good example from my seer project:


```
$font-sizes: ( 
xs:   1rem,       // 16px
sm:   1.0625rem,  // 17px
md:   1.1875rem,  // 19px
lg:   2.1875rem,  // 35px
xl:   2.5rem,     // 40px
)
```